rezervo/chains/schema.py

```diff
@@ -0,0 +1,28 @@
+from pydantic import BaseModel
+
+from rezervo.providers.schema import Branch
+
+
+class ThemeSpecificImages(BaseModel):
+    largeLogo: str
```

Comment:
Here we can use `CamelModel` to avoid cursing the codebase with camels :camel: :upside_down_face:

Reply:
🐫